packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -79,6 +79,7 @@ const checkboxStyles = css`
   ${boxSizingStyles};
   ${({ size }) => styles[size].baseFontSize};
   line-height: ${({ hasLabel }) => (hasLabel ? lineHeightValue(-1) : "0")};
+  margin: 0 0 ${tokens.space} 0;
```

Comment:
Adding a margin below checkbox and radio component by default as it will typically be used in this way.
![image](https://user-images.githubusercontent.com/4040102/73216354-b3308000-410a-11ea-8132-75bd461537d8.png)